core/src/test/java/io/confluent/kafka/schemaregistry/client/CachedSchemaRegistryClientTest.java

```diff
@@ -154,6 +171,21 @@ public void testAvroProducer() {
     assertArrayEquals(objects, recordList.toArray());
   }
 
+  @Test
+  public void testAvroProducerWithCompositeDeserializer() {
+    String topic = "testAvro";
+    IndexedRecord avroRecord = createAvroRecord();
+    Object[] objects = new Object[]{avroRecord};
```

Comment:
ideally we want to produce two records, no? one using the old and another using the new Serializers

Reply:
Ok, let me take a look